### DISCUSSION TITLE
参考資料求む

### DISCUSSION THREAD

**Discussion by hideojoho** [ir100/ir100#5] [Ideas]
## ゴール
- コンテンツ作成の参考になる資料を幅広く集約する
- `References.md`というファイルに追加していく

## 動機
- コンテンツの範囲、構成、難易度（？）に関する議論の土台としたい

## 例
- [言語処理100本ノック](https://nlp100.github.io/ja/)

**Comment by tmanabe** [comment]
パッと思いついたのはこれですね。
https://qiita.com/KoheiShinden/items/7bcf5017195177591170#1-%E6%83%85%E5%A0%B1%E6%A4%9C%E7%B4%A2%E3%81%AE%E5%9F%BA%E6%9C%AC%E7%9A%84%E3%81%AA%E5%85%A8%E4%BD%93%E5%83%8F%E3%82%92%E7%9F%A5%E3%82%8B

**Comment by kohei-shinden** [comment]
Python の実装という文脈だと、[Terrier チーム](http://terrier.org/) が ECIR 2021 でやっていたチュートリアルは Bag-of-words から Dense retrieval までをいくつかの手法をピックアップしてコードを提示しているのでかなり参考になるかなと思いました。この資料が英語の処理なので日本語の処理に変更するという感じで。ただ、どの検索ライブラリ（PyTerrier, Pyserini）を使うかというのは要検討だなと思います。

https://github.com/terrier-org/ecir2021tutorial
